### PR TITLE
Describe graceful deletionTimestamp more accurately

### PR DESCRIPTION
**Pull request by smarterclayton**
Spawned from #34160

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/35481)

### CODE REVIEW

File: pkg/api/types.go [hunk 1]
```diff
@@ -114,15 +114,23 @@ type ObjectMeta struct {
 	// +optional
 	CreationTimestamp unversioned.Time `json:"creationTimestamp,omitempty"`
 
-	// DeletionTimestamp is the time after which this resource will be deleted. This
+	// DeletionTimestamp is RFC 3339 date and time at which this resource will be deleted. This
```
Comment (kargakis) [inline pkg/api/types.go]: s/will be deleted/is marked for deletion/

Comment (smarterclayton) [inline pkg/api/types.go]: I don't think that's the right change.

**Comment by smarterclayton** [comment]
Any other comments on this description?

**Comment by davidopp** [comment]
LGTM

**Comment by smarterclayton** [comment]
```
I1103 08:59:54.850] >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
I1103 08:59:54.850] >                              START TEST                                >
I1103 08:59:54.850] >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
I1103 08:59:54.850] Start Test Suite on Host 
I1103 08:59:54.850] 
I1103 08:59:54.850] Failure Finished Test Suite on Host 
I1103 08:59:54.851] unable to create gce instance with running docker daemon for image e2e-node-ubuntu-trusty-docker9-v1-image.  instance tmp-node-e2e-0b58bc68-e2e-node-ubuntu-trusty-docker9-v1-image not running docker daemon - Command failed: Permission denied (publickey).
```

**Comment by smarterclayton** [comment]
@k8s-bot node e2e test this

**Comment by smarterclayton** [comment]
Doc rebase

**Pull request merged**